intuita-webview/src/codemodList/TreeView/Tree.tsx

```diff
@@ -3,6 +3,19 @@ import { ReactNode } from 'react';
 import { CodemodHash, CodemodTreeNode } from '../../shared/types';
 import { useKey } from '../../jobDiffView/hooks/useKey';
 
+const getIndex = (
+	nodes: ReadonlyArray<CodemodTreeNode>,
+	hash: CodemodHash | null,
+): number | null => {
+	if (hash === null) {
+		return null;
+	}
+
+	const indexInNodesByDepth =
```

Comment:
1) return immediately in this line
2) findIndex doesn't yield `null`, but `-1` in case of a problem